tests/snippets/builtin_enumerate.py

```diff
@@ -5,6 +5,19 @@
 assert list(enumerate(['a', 'b', 'c'], -100)) == [(-100, 'a'), (-99, 'b'), (-98, 'c')]
 assert list(enumerate(['a', 'b', 'c'], 2**200)) == [(2**200, 'a'), (2**200 + 1, 'b'), (2**200 + 2, 'c')]
 
+a = list([1, 2, 3])
```

Comment:
`[1, 2, 3]` is already a list?

Reply:
Also, why is this in `enumerate` tests file?

Reply:
Yes, it a list already. I moved the tests to `list.py`.